ui/aboutPC.html

```diff
@@ -1,23 +1,5982 @@
-<!-- 
-Version : Phython/Django 2.7.6, PostgreSQL 9.3.4
-Author : Vaibhavi Desai
-Github username : desaivaibhavi
-email : ranihaileydesai@gmail.com
--->
-
-<!-- page for about us -->
-<html>
-<head>
-	<title> About PC </title>
-</head>
-<body>
-    {% include "header.html" %}
-    <br><br><br><br><br><br><center>
-    <p>Here the information about PC would be shown</p></center>
-    
-    <br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br><br>
-    {% include "footer.html" %}
-    
-</body>
-</html>
-
+<!-- 
+Version : Phython/Django 2.7.6, PostgreSQL 9.3.4
+Author : Vaibhavi Desai
+Github username : desaivaibhavi
+email : ranihaileydesai@gmail.com
+-->
+
+<!-- page for about us -->
+<html>
+    <head>
+        <title> About PC </title>
+
+        <style>

```

Comment:
Instead of a <style> element, you can include a <link> element inside the <head> and add a reference to the bootstrap stylesheet. 

Something like this ```<link rel="stylesheet" type="text/css" href="css/bootstrap.min.css">```